hardhat.config.ts

```diff
@@ -6,8 +6,33 @@ import 'dotenv/config';
 import type { HardhatUserConfig } from 'hardhat/config';
 
 const config: HardhatUserConfig = {
-  etherscan: hardhatConfig.etherscan(),
-  networks: hardhatConfig.networks(),
+  // NOTE: According to https://docs.sei.io/evm/evm-verify-contracts#config-file
+  etherscan: {
+    ...hardhatConfig.etherscan(),
+    apiKey: {
```

Comment:
I'm surprised to see this is not already supported by [@api3/contracts](https://github.com/api3dao/contracts/blob/main/src/hardhat-config.ts#L59) package 🤔

Reply:
I was as well, and when I displayed the list of supported networks SEI wasn't one of them so I opted for this way.